dask/dataframe/_compat.py

```diff
@@ -181,3 +181,9 @@ def is_any_real_numeric_dtype(arr_or_dtype) -> bool:
             and not is_complex_dtype(arr_or_dtype)
             and not is_bool_dtype(arr_or_dtype)
         )
+
+
+def is_arrow_dtype(dtype) -> bool:
```

Comment:
Is there similar-ish utility in `pandas` we could use? What you have here is fine, I'm just curious

Reply:
No, we check explicitly in pandas since we don't have to worry about different versions.